homeassistant/components/vesync/fan.py

```diff
@@ -174,6 +174,9 @@ def extra_state_attributes(self):
         if hasattr(self.smartfan, "air_quality"):
             attr["air_quality"] = self.smartfan.air_quality
 
+        if self.smartfan.details.get("air_quality_value") is not None:
+            attr["air_quality_value"] = self.smartfan.details["air_quality"]
```

Comment:
```suggestion
            attr["air_quality_value"] = self.smartfan.details["air_quality_value"]
```

Reply:
I was actually up-in-the-air about this as well. I considered renaming the value that was added recently `air_quality_simple` and making `air_quality` the pm2.5 value again (like it was in previous releases), but that seemed like it might be confusing for people working on `pyvesync`.

I also considered naming the pm2.5 value `pm25` (or similar) in order to make it more clear what it actually represented. Definitely open to suggestions there, though.

Reply:
Maybe `particulate_matter_2_5` to match the entity that it will likely eventually correspond to?

Reply:
For what it's worth, I think the new `air_quality` metric has value here as well, because I think a few of the supported fans have some sort of vague air quality measurement that isn't pm2.5-based and doesn't report raw sensor value (specifically, the `LV-PUR131S` seems to report with strings such as `Excellent`). I think that is why it was added/changed in `pyvesync` to begin with, but I only have a 400s (pm2.5 sensor) and a 200s (no air quality measurement at all), so I can't really confirm that that is the case or whether it is useful and am only working off of assumptions from reading the `pyvesync` source and documentation.

Reply:
I'm more than happy to swap this again, but I did a quick revision using `pm25` as the attribute name and deployed that change to my home instance to make sure everything was working as expected. Here are my attributes locally with this patch in place:
![image](https://user-images.githubusercontent.com/3813932/168320082-c0aa1369-c4a4-4b86-872c-4467d5938aaa.png)

Reply:
I think we should just restore the previous behavior since we want to remove these attributes in the future and expose the measurements as individual sensors instead.

Reply:
Sounds fair enough to me, coming right up 👍

Reply:
Alright, from the perspective of a Home Assistant user everything is now back to exactly the way it was before :)
![image](https://user-images.githubusercontent.com/3813932/168325015-979cb210-220a-4af6-a452-4c95f2303c92.png)